content/en/docs/concepts/storage/volumes.md

```diff
@@ -34,8 +34,9 @@ Kubernetes supports many types of volumes. A {{< glossary_tooltip term_id="pod"
 can use any number of volume types simultaneously.
 Ephemeral volume types have a lifetime of a pod, but persistent volumes exist beyond
 the lifetime of a pod. Consequently, a volume outlives any containers
-that run within the pod, and data is preserved across container restarts. When a
-pod ceases to exist, the volume is destroyed.
+that run within the pod, and data is preserved across container restarts. When a pod
+ceases to exist, Kubernetes destroys ephemeral volumes; however, Kubernetes does not
+destroy persistent volumes.
```

Comment:
Suggestion: 

Ephemeral volume types have the lifespan of a Pod. Persistent volume types exist past the lifespan of a Pod. as a result, volumes outlive containers running in the Pod, preserving data across container restarts. When the Pod ceases to exist, Kubernetes destroys ephemeral volumes, but does not destroy persistent volumes.

Reply:
@jailton , Would you rebase your changes into a single commit? Thanks!
@shannonxtreme Thanks for the suggested edits. What do you think about opening a new PR with these changes?

Reply:
Done.